templates/includes/DocumentCart_email.ss

```diff
@@ -0,0 +1,45 @@
+<div>
+    <% include RequestItems_email %>
+    <table>
+        <thead>
+        <tr>
+            <th colspan="2">
+                <%t DMSCart.DELIVERY_INFORMATION "Delivery Information" %>

```

Comment:
Fine for now - for future reference translations should be bound against an actual class name. In SS4 this is a requirement - in SS3 it's less strict.